decidim-core/lib/decidim/participatory_space.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This concern contains the logic related to participatory spaces.
+  module ParticipatorySpace
+    extend ActiveSupport::Concern
+
+    included do
+      def manifest
+        self.class.participatory_space_manifest
+      end
+    end
+
+    class_methods do
+      def participatory_space_manifest
+        Decidim.find_participatory_space_manifest(name.demodulize.underscore.pluralize)
+      end
+    end
+  end
+end
```

Comment:
This concern and "participable.rb" should probably be merged together? Also, you can probably find `name.demodulize.underscore.pluralize` in a method over there :)

Reply:
Ah yeah, sure! I'll merge them :)